lib/plurimath/math/function/table.rb

```diff
@@ -210,6 +210,30 @@ def norm_table(table_tag)
             [mo_tag, table_tag, mo_tag],
           )
         end
+
+        def fenced_table(ox_table)
+          return ox_table unless open_paren && close_paren
+
+          d_node = Utility.ox_element("d", namespace: "m")
+          e_node = Utility.ox_element("e", namespace: "m")
+          e_node << ox_table
+          Utility.update_nodes(d_node, [mdpr_node, e_node])
+        end
+
+        def mdpr_node
+          begchr_attr = { "m:val": open_paren } if open_paren
+          endchr_attr = { "m:val": close_paren } if close_paren
+          sepchr_attr = { "m:val": "" }
+          begchr = Utility.ox_element("begChr", attributes: begchr_attr, namespace: "m")
```

Comment:
I find it strange that these variables may not even be set when being used here. There must be better ways to do this...